plf-packaging-resources/src/main/resources/CHANGE_LOG.txt

```diff
@@ -3,318 +3,172 @@ Changelog Platform ${org.exoplatform.platform.version}
+** Improvements
+    * CAL-1226 - Make URLs in event descriptions clickable
 
-** New Feature
-    * [SOC-4928] - [Social Activity Addon] Implements StreamFilter for Feed, Connections, My Spaces and Owner
-    * [SOC-4929] - [Social Activity Addon] Implements get Older and Newer for StreamFilter for Feed, Connections, My Spaces and Owner
-    * [SOC-4930] - [Social Activity Addon] Implements CRUD Comment and updates the activity.
-    * [SOC-4931] - [Social Activity Addon] Apply the liquibase to manage the DB structure
-            
+** Bugs
+    * CAL-1193 - [Repetitive Event] Wrong date when starting time is in previous day in GMT
+    * CAL-1215 - Tests failures with JDK 8
+    * CAL-1221 - Fix javadoc errors raised by doclint (JDK 8)
+    * CAL-1224 - Bad translation for Location in french
+    * CAL-1225 - User picker in Participants tab is case sensitive
+    * CAL-1237 - Wrong From fied's value in a calendar reminder email notification after open relay mail configuration
 
-eXo Calendar 4.3.0-RC1  
+eXo Integration ${org.exoplatform.platform.version}
 *************************
 
-** Bug
-    * [CAL-705] - Cannot resize an event with duration of 30 minutes in Week view
-    * [CAL-1122] - Request a ICS calendar with no event/tasks, nothing return
-    * [CAL-1123] - When the field groups is fill, no one is owner
-    * [CAL-1136] - Impossible to edit the field " groups "
-    * [CAL-1137] - When john wants to edit the owner of the calendar the error code is wrong
-    * [CAL-1147] - No reminder pop-up when Platform uses Apache web server
-    * [CAL-1155] - CalendarRestApi sub-resources are wrong
-    * [CAL-1157] - CalendarRestApi - null attachments
-    * [CAL-1175] - CalendarRestApi - returned task status is always empty
-    * [CAL-1184] - Can not export Personal Calendar
-    * [CAL-1188] - Cannot delete a personal/group/shared calendar 
-    * [CAL-1189] - Unknown error when click view on deleted event
-
-** Improvement
-    * [CAL-1083] - Reminder mails are always in English
-
-
-eXo Integration 4.3.0-RC1
+** Bugs
+    * INTEG-451 - Fix javadoc errors raised by doclint (JDK 8)
+    * INTEG-457 - [Unified Search] Defaut sort by date is made from the most old to the most recent
+    * INTEG-458 - [Unified search] File not found when it contains "_"
+    * INTEG-463 - org.exoplatform.commons.search.driver.jcr.JcrSearchDriver problem in replaceSpecialCharacters
+
+eXo Forum ${org.exoplatform.platform.version}
 *************************
 
-** Bug
-    * [INTEG-417] - Avoid systemComment variable on postActivity of org.exoplatform.wcm.ext.component.activity.listener.Utils
-    * [INTEG-423] - [integ-ecms] Comment when updating summary for content is not right
-    * [INTEG-426] - wrong position of x icon
-    * [INTEG-432] - [Unified Search] Missing content type "Task" in the Filter
-    * [INTEG-435] - Bad display of Recent Activities in my profil
+** Improvements
+    * PFR-1116 - Forum Redesign
```

Comment:
overstatement. a more accurate summary would be : [Forum] Mobile friendly topic list